docs/user_guide/outputs/jetstream_output.md

```diff
@@ -123,6 +123,10 @@ outputs:
     # boolean, enables extra logging for the nats output
     debug: false
     # boolean, enables the collection and export (via prometheus) of output specific metrics
+    # integer, sets the size of the local buffer where received
+    # NATS messages are stored before being sent to outputs.
+    # This value is set per worker. Defaults to 100 messages
+    buffer-size: 100
```

Comment:
Looks like this was inserted after the comment for `enable-metrics: false`.
Can we also keep the default to 0 to not change the default behavior?
I didn't see the default being set in code anyways.

Reply:
Certainly, I can make that change

Reply:
This has been updated as requested.